packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -260,6 +261,17 @@ function willDrunkAdventure() {
   return have($item`Drunkula's wineglass`) && globalOptions.ascend;
 }
 
+function canGetFusedFuse() {
+  return (
+    realmAvailable("hot") &&
+    ([1, 2, 3] as const).some(
+      (it) => get(`_volcanoItem${it}`) === $item`fused fuse`.id,
+    ) &&
+    !globalOptions.clarasBellClaimed &&
```

Comment:
won't globalOptions.clarasBellClaimed be true here because it's set in checkVolcanoQuest()?